parm/config/gefs/config.stage_ic

```diff
@@ -7,32 +7,24 @@ echo "BEGIN: config.stage_ic"
 # Get task specific resources
 source "${EXPDIR}/config.resources" stage_ic
 
-case "${CASE}" in
-  "C384")
-    export CPL_ATMIC=""
-    export CPL_ICEIC=""
-    export CPL_OCNIC=""
-    export CPL_WAVIC=""
-    export CPL_MEDIC=""
-    ;;
-  "C96")
-    export CPL_ATMIC=""
-    export CPL_ICEIC=""
-    export CPL_OCNIC=""
-    export CPL_WAVIC=""
-    export CPL_MEDIC=""
-    ;;
-  "C48")
-    export CPL_ATMIC="gefs_test"
-    export CPL_ICEIC="gefs_test"
-    export CPL_OCNIC="gefs_test"
-    export CPL_WAVIC="gefs_test"
-    export CPL_MEDIC="gefs_test"
-    ;;
-  *)
-    echo "FATAL ERROR Unrecognized resolution: ${CASE}"
-    exit 1
-    ;;
-esac
+export STAGE_IC_YAML_TMPL="${PARMgfs}/stage/stage.yaml.j2"
+
+# Set ICSDIR
+
+if [[ -z "${ICSDIR}" ]] ; then
+
+  IC_VER="20240610"
```

Comment:
Please make the non-exported variables lowercase.

Reply:
Done @ fb58d6d, thanks!